hw/ip/csrng/doc/checklist.md

```diff
@@ -179,25 +179,25 @@ Review        | [V2_CHECKLIST_SCOPED][]               | Done        |
 
  Type         | Item                                    | Resolution  | Note/Collaterals
 --------------|-----------------------------------------|-------------|------------------
-Documentation | [DESIGN_DELTAS_CAPTURED_V2][]           | Not Started |
-Documentation | [DV_DOC_COMPLETED][]                    | Not Started |
-Testbench     | [FUNCTIONAL_COVERAGE_IMPLEMENTED][]     | Not Started |
-Testbench     | [ALL_INTERFACES_EXERCISED][]            | Not Started |
+Documentation | [DESIGN_DELTAS_CAPTURED_V2][]           | Done        |
+Documentation | [DV_DOC_COMPLETED][]                    | Done        |
```

Comment:
can you fix the format problem and broken links in section `Global types & methods`.
https://docs.opentitan.org/hw/ip/edn/doc/dv/index.html

Reply:
Done

Reply:
the doc seems not fixed yet. Is the fix in a pending PR?

Reply:
No, this one. I fixed one issue by adding an example of a parameter to the "Global types and Methods" section. @weicaiyang Is there something else?